impeller/renderer/backend/vulkan/pass_bindings_cache.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <optional>
+
+#include "flutter/impeller/renderer/backend/vulkan/vk.h"
+
+namespace impeller {
+
+class PassBindingsCache {
+ public:
+  void bindPipeline(vk::CommandBuffer command_buffer,
```

Comment:
Nit: `CapitalCase`.

Reply:
I only did it so that they match verbatim the functions they are caching for (ie `vk::CommandBuffer::bindPipeline`).

Reply:
I like being consistent in newly authored code. And besides, the stuff in libc++ is `underscore_case`. So having different conventions for code and calls in different components is a thing already.

In any case, I don't have a strong opinion about this. But if you are ambivalent about it, I'd rather go with sticking to the style guide.

Reply:
Done